apps/src/lib/util/AnalyticsReporter.js

```diff
@@ -10,10 +10,13 @@ import {
   getEnvironment,
   isProductionEnvironment,
   isStagingEnvironment,
+  isDevelopmentEnvironment,
 } from '../../utils';
 
 // A flag that can be toggled to send events regardless of environment
 const ALWAYS_SEND = false;
+// A flag that can be toggled to console log events in development
+const LOG_EVENT_IN_DEV = false;
```

Comment:
Curious why @rshipp thinks here but I would still prefer these to be automatically logged in development only. If I'm the only one, though, I think this is okay.

Also tagging @hannahbergam as she's been working with Amplitude a lot in the last few months.

Reply:
Agreed! Love having those local logs but would be happy to see them removed from Test.